<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>1307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="parent_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-14</y>
     <width>1251</width>
     <height>1031</height>
    </rect>
   </property>
   <widget class="QWidget" name="z_vs_x_widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>711</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>681</width>
      <height>671</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>671</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="crosshair_widget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="crosshairButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="text">
        <string>Crosshair</string>
       </property>
       <property name="iconSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="crosshair_width"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="crosshair_x"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="crosshair_y"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>step</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clear_crosshair_Button">
       <property name="text">
        <string>Clear Crosshair</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="center_Button">
       <property name="text">
        <string>center</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="crosshair_step"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="z_vs_y_widget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>331</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
